.golangci.yml

```diff
@@ -11,15 +12,29 @@ output:
   sort-results: true
 
 linters-settings:
+  depguard:
+    include-go-root: true
+    packages:
+      - flag
+      - io/ioutil
+      # - reflect # TODO: Disallow reflect package
+      - unsafe
```

Comment:
the performance and speed of fiber and fasthttp relies on some trick with the unsafe packet
i am not sure if we should limit this ? or is the plan to make exceptions where this is used ?

Reply:
I'd go with a allowlist-approach here, i.e. deny the `unsafe` package by default and require `//nolint:depguard //  Explanation why an import of the "unsafe" package is required here`.

Reply:
ok